dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -435,8 +434,9 @@ private void bulkUpsertWorkflows(User authUser) {
 
         // TODO: Only update the ones that have changed
         authUser = userDAO.findById(authUser.getId());
+        List<Tool> finalTools = getTools(authUser);
```

Comment:
Swagger value should note that the method returns all repos, not just the ones that were refreshed